en/micro-integrator/docs/references/connectors/file-connector/file-connector-example.md

```diff
@@ -0,0 +1,138 @@
+# File Connector Example
+
+The File Connector allows you to connect to different file systems and perform various operations with the file systems. The file connector uses the Apache Commons VFS I/O functionalities to execute operations.
```

Comment:
```suggestion
The File Connector allows you to connect to different file systems and perform various operations. The File Connector uses the Apache Commons VFS I/O functionalities to execute operations.
```

Let's be consistent with connector names and write them in title case everywhere. So use "File Connector" in all instances.

Reply:
Removed the 'File Connector', since we are working in a generic document.

Reply:
Done

Reply:
Done